AlbumList/AlbumListViewController.swift

```diff
@@ -0,0 +1,187 @@
+//
+//  AlbumListViewController.swift
+//  AlbumList
+//
+//  Created by tommy tayler on 28/01/23.
+//
+
+import UIKit
+
+var albumTitle = ""
```

Comment:
Best to avoid global variables where possible. Lets go with a view model approach and for sub pages like Thumbnail View Controller, pass in the variables required when we initialise it